samples/emoji-search/presenters/src/jsMain/kotlin/app/cash/zipline/samples/emojisearch/RealEmojiSearchPresenter.kt

```diff
@@ -43,15 +42,15 @@ class RealEmojiSearchPresenter(
 
   override fun launch(): ZiplineTreehouseUi {
     val events = MutableSharedFlow<EmojiSearchEvent>(extraBufferCapacity = Int.MAX_VALUE)
-    val factories = EmojiSearchDiffProducingWidgetFactories(ProtocolBridge(), json)
+    val bridge = EmojiSearchProtocolBridge.create(json)
```

Comment:
This also makes "diff-producing" a thing that is no longer in public APIs which is a nice win and that I'm eager to see for "diff-consuming".